### DISCUSSION TITLE
GitHub Copilot could not connect to server. Extension activation failed - Behind Proxy

### DISCUSSION THREAD

**Discussion by johnschult** [community/community#33279] [Copilot Conversations]
I get the following error when the Copilot extension starts up:

```
[ERROR] [default] [2022-09-16T16:40:25.601Z] GitHub Copilot could not connect to server. Extension activation failed: 
"4482075544:error:10000129:SSL routines:OPENSSL_internal:TLS13_DOWNGRADE:../../third_party/boringssl/src/ssl/handshake_client.cc:763:"
```

I am using a Mac and am running behind a corporate proxy, however I never have any other problems connecting to anything. I can ping `api.github.com` just fine.

**Comment by iamcjwilliams** [comment]
@johnschult Have you upgraded your extension? Copilot 1.44.0.0 VS extension was just released and contain various improvements that help address issues on VS 17.3.x
